extensions/mssql/src/mssql.d.ts

```diff
@@ -1379,6 +1380,22 @@ declare module 'mssql' {
 			containmentTypes: string[];
 			recoveryModels: string[];
 		}
+
+		export interface PropertiesInfo extends SqlObject {
```

Comment:
Shouldn't this be something like `ServerPropertiesInfo`?

Reply:
As per the comment below, it seems the interface is for the common properties that exists in Server and Database properties list.